vector/api/endpoints/fetch.mdx

```diff
@@ -49,7 +49,10 @@ their vector ids.
       The id of the vector.
     </ResponseField>
     <ResponseField name="vector" type="number[]">
-      The vector value.
+      The dense vector value for dense and hybrid indexes.
+    </ResponseField>
+    <ResponseField name="sparseVector" type="Object[]">
```

Comment:
```suggestion
    <ResponseField name="sparseVector" type="number[]">
```

Reply:
Ah sorry, then can we use something like expandables to specify the object's fields

https://mintlify.com/docs/content/components/expandables

Reply:
https://github.com/upstash/docs/pull/346/commits/31b8ef474bd20e82cde7ffa53d9d65b209e38d18